frontend/assets/bayern/translations/de.json

```diff
@@ -0,0 +1,130 @@
+{
+  "publisher": "Herausgeber",
```

Comment:
@f1sh1918 @sarahsporck @michael-markl do you think it makes sense to streamline these constants here and perhaps introduce namespaces? I would stick close to the folder structure and use `about`, `identification`, `map`, `navigation`, `permission`, `search`, `store` and `category`. What do you think?

Reply:
Yes i think namespaces would be good to gain a better overview. I think you can do it here